ideas/145-identity.md

```diff
@@ -0,0 +1,147 @@
+
+#### Fundamental terms:
+
+- ID Manager (owner) can define MultiSig public keys required for its management/actions;
+- ID Manager (owner) can define a Recovery contract;
+- ID Manager (owner) can define public keys for other; users knowing how to encrypt data for identity owner;
+- ID Manager (owner) can accept claims;
+- ID Manager (owner) can make claims.
+
+#### Status Use Case:
+
+From user perspective, instead of talking to obscure address/contracts, users  would able to confirm that a person is whom they expect to be interacting.
+Mainly because they usually know by community network who is the Identity holder and can safely assume is the right person.
+
+This is important for chat system, where Identities provide the right key for others messaging, and Identity owners can always update this keys as they want.
+
+Recovery contract, if safely defined, will garantee that Identity owner is able to get back their Identity from loss or theft, therefore no need to create a new one and ask friends to update their contact list with your new Identity.
+
+#### External references: 
+
+- [Fabian Vogelsteller - ERC: Identity - Ethereum London](https://www.youtube.com/watch?v=jv3BmGGFP7c)
+- [DEVCON3: ERC Identity](https://www.youtube.com/watch?v=pkwYVagytuA)
+- [Jolocom: enabling blockchain agnostic self- sovereign identity - Kai Wagner](https://www.youtube.com/watch?v=vkEdDj5HtVs)
+- [Mist: UX insight - Alexande Van de Sande](https://gist.github.com/alexvandesande/434f143fc6d08cb4388479a3d9f527a9)
+- [DEVCON1: Digital Identity - Christian Lundkvist](https://www.youtube.com/watch?v=QpaTOvAhLR4)
+- [uPort – Usable key management & identity - Rouven Heck and Dr. Christian Lundkvist](https://www.youtube.com/watch?v=qRevDM9D8WE)
+- [ConsenSys and uPort: Powering Decentralized Identity](https://www.youtube.com/watch?v=VXAZdBtN3N0)
+- [Self-sovereign BigchainDB data injection in smart contracts through the Jolocom Identity Gateway](https://www.youtube.com/watch?v=8K-BDlsx8KQ)
+- [IDbox – Cost efficient device for self-sovereign identity](https://www.youtube.com/watch?v=h1Oz3oEtZxE)
+
+### Product Description
+
+- Smart contracts: See [GDocs Identity Terms Intent](https://docs.google.com/document/d/1K8tFjGneScuKiudpD3HfSpAd4eOe8012jn0Q3iDxEV0/edit#) and [Identity Readme](https://github.com/status-im/contracts/blob/develop/Identity.md)
+- Basic contracts demo: Helps engaging developers and bug hunting. 
+- Identity Sign-in: Actions for user creating or recovering their wallets.
+- Identity Usage: User may select their Identity to make tansactions in behalf of.
+- Identity Management: User may setup keys & recovery options for its identity, confirm claims and even update the contract code.
+
+### Requirements & Dependencies
+
+- ERC725 and ERC735 should be approved as standard and used in contracts interface, this swarm might influence how this standards end up defined, this is important for the solution working better with unknown future applications.
```

Comment:
For each of these requirements, it'd be useful to understand at what point they are blockers

Reply:
Identity itself is independent, the requirements are mostly for better UX and ERCs for developer community.